index.html

```diff
@@ -27,7 +27,7 @@
           </ul>
         </div>
         <div class="mobile-menu-icon">
-          <img src="./images/Union.svg" alt="">
+          <img src="./images/Union.svg" alt="Bar Image">
```

Comment:
- [ ] You have modified the `Page title and
Image text alternatives ` but your report in the pull request does not contain the link to the modifications, as the requirement state below
<img width="851" alt="1" src="https://github.com/newtonah0/Mobile-Portfolio/assets/77843050/745538c5-f387-4f25-855f-dfdf4b5ad52f">